estate-services/src/main/java/org/egov/ps/service/LocalisationService.java

```diff
@@ -0,0 +1,65 @@
+package org.egov.ps.service;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.jayway.jsonpath.JsonPath;
+
+import org.egov.common.contract.request.RequestInfo;
+import org.egov.ps.config.Configuration;
+import org.egov.ps.repository.ServiceRequestRepository;
+import org.egov.ps.util.PSConstants;
+import org.egov.ps.web.contracts.RequestInfoWrapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Service
+
+public class LocalisationService {
+    @Autowired
+	private Configuration config;
+
+	@Autowired
+    private ServiceRequestRepository serviceRequestRepository;
+    
+    public static final Map<String, Map<String, String>> localisedMessageMap = new HashMap<>();
+/**
+	 * Populates the localized msg cache
+	 * 
+	 * @param requestInfo
+	 * @param tenantId
+	 * @param locale
+	 * @param module
+	 */
+	public void getLocalisedMessages(RequestInfo requestInfo, String tenantId, String locale, String module) {
```

Comment:
This method should return the map right away.
We can however cache based on the tenantId, locale and module using `@Cacheable`. https://spring.io/guides/gs/caching/